apps/trustlab/src/payload/blocks/ResearchCategory.js

```diff
@@ -54,6 +54,39 @@ const ResearchCategory = {
         },
       ],
     },
+    {
+      name: "settings",
+      type: "group",
+      label: {
+        en: "Single Report Settings",
+      },
+      fields: [
+        {
+          name: "backButtonText",
```

Comment:
```suggestion
          name: "backButtonLabel",
```

You seem to be using "label" in download, overview, etc. Why changing here?